app/create_app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, render_template, request, redirect
+from app.trello_client import TrelloClient
+from app.viewmodel import ViewModel
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object('app.flask_config.Config')
+    client = TrelloClient()
+
+    @app.route('/', methods=['GET'])
+    def index():
+        sorted_items = sorted(client.get_items(), key=lambda item: item.status, reverse=True)
```

Comment:
Very minor point, but I'd say the sorting belongs in the ViewModel conceptually (and would be easier to re-use in a larger app)

Reply:
Sorted - 36d6ef4695a47a488f458543a0477c4e307ae3a2